### DISCUSSION TITLE
Using a different font only when rendered

### DISCUSSION THREAD

**Discussion by ertwro** [jbyuki/nabla.nvim#58] [Ideas]
Hi. I've found that a font like Cambria Math does a better job of rendering math in plan text. It would be pretty cool if one could use a different font only for text when is rendered

**Comment by jbyuki** [comment]
There are a few fonts that are capable of rendering unicode math symbols, I suggest looking for one where the text looks nice and the math rendering is also nice. This restrict the choice of fonts but on the other hand, it's also the whole point of rendering math with unicode. 

While it would be nice to mix two fonts together, neovim and more generally terminals do not allow such a feature, as far as I'm aware. Maybe some do but it would be a hassle to support because ideally it should support any platform that runs neovim which it does to some degree. If we mix fonts, it's a bit the same idea as mixing with images, which I don't plan to support here either because of the same reasons.

But thank you for the suggestion, I might look into it if something comes up under my radar. I noticed a few users complaining about the font rendering which is definitely something to point out.
